libs/sdk-ui-dashboard/src/presentation/filterBar/attributeFilter/CustomDropdownActions.tsx

```diff
@@ -0,0 +1,107 @@
+import {
+    AttributeFilterConfigurationButton,
+    AttributeFilterDeleteButton,
+} from "@gooddata/sdk-ui-filters/esm/internal";
+import { selectIsDeleteFilterButtonEnabled, selectIsInEditMode, useDashboardSelector } from "../../../model";
+
+/**
+ * @internal
+ */
+export interface ICustomAttributeFilterDropdownActionsProps {
+    onApplyButtonClick: () => void;
+    onCancelButtonClick: () => void;
+    onConfigurationButtonClick: () => void;
+    onDeleteButtonClick: () => void;
+    isApplyDisabled?: boolean;
+    cancelText: string;
+    applyText: string;
+}
+
+/**
+ * @internal
+ */
+export const CustomAttributeFilterDropdownActions: React.VFC<ICustomAttributeFilterDropdownActionsProps> = ({
+    isApplyDisabled,
+    onApplyButtonClick,
+    onCancelButtonClick,
+    onConfigurationButtonClick,
+    onDeleteButtonClick,
+    cancelText,
+    applyText,
+}) => {
+    const isEditMode = useDashboardSelector(selectIsInEditMode);
+    const isDeleteButtonEnabled = useDashboardSelector(selectIsDeleteFilterButtonEnabled);
+
+    return (
+        <div className="gd-attribute-filter-dropdown-actions__next">
+            <div className="gd-attribute-filter-dropdown-actions-left-content__next">
+                {isEditMode && isDeleteButtonEnabled && (
+                    <>
+                        <AttributeFilterDeleteButton onDelete={onDeleteButtonClick} />
+                        <div className="gd-button-separator" />
+                    </>
+                )}
+                {isEditMode && (
+                    <AttributeFilterConfigurationButton onConfiguration={onConfigurationButtonClick} />
+                )}
+            </div>
+            <div className="gd-attribute-filter-dropdown-actions-right-content__next">
+                <Button
+                    className="gd-attribute-filter-cancel-button__next gd-button-secondary gd-button-small cancel-button s-cancel"
+                    onClick={onCancelButtonClick}
+                    value={cancelText}
+                    title={cancelText}
+                />
+                <Button
+                    disabled={isApplyDisabled}
+                    className="gd-attribute-filter-apply-button__next gd-button-action gd-button-small s-apply"
+                    onClick={onApplyButtonClick}
+                    value={applyText}
+                    title={applyText}
+                />
+            </div>
+        </div>
+    );
+};
+
+/**
+ * @internal
+ */
+export interface ICustomConfigureAttributeFilterDropdownActionsProps {
+    onSaveButtonClick: () => void;
+    onCancelButtonClick: () => void;
+    isSaveDisabled?: boolean;
+    cancelText: string;
+    saveText: string;
+}
+
+/**
+ * @internal
+ */
+export const CustomConfigureAttributeFilterDropdownActions: React.VFC<
+    ICustomConfigureAttributeFilterDropdownActionsProps
+> = ({ isSaveDisabled, onSaveButtonClick, onCancelButtonClick, cancelText, saveText }) => {
+    return (
```

Comment:
Its strange that eslint is not complaining about Display mode in this inline component